### PR TITLE
CHANGELOG.md: prepare for release 0.1.1

### PR DESCRIPTION
**Pull request by patrislav**

### CODE REVIEW

**Comment by xescugc** [comment]
Follow this format https://github.com/cycloidio/inframap/blob/master/CHANGELOG.md

**Comment by patrislav** [comment]
Changed to follow the given format, also rebased since the changes are minimal.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,7 @@
-# Changelog
-All notable changes to this project will be documented in this file.
+## [Unreleased]
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.1.1] - _2021-04-29_
```
Comment (xescugc) [inline CHANGELOG.md:4]: Remove the `-` just to have the same format on all OSS projects :)

**Review by xescugc — COMMENTED** [review]
Single comment.

**Comment by patrislav** [comment]
Fixed

**Review by xescugc — APPROVED** [review]

**Pull request merged**